### PR TITLE
Add conda/conda-build#3868

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by msarahan** [comment]
These are changes that need to be approved on conda build, not as patches here.

**Comment by isuruf** [comment]
I've made a PR in conda-build. I don't see how this is different from when we patch other packages like `python`.

**Comment by scopatz** [comment]
Yeah, I agree with @isuruf - and I trust him to keep this up-to-date

**Comment by jjhelmus** [comment]
> I don't see how this is different from when we patch other packages like `python`.

The patches we apply to Python allow it to work with the conda ecosystem.  These introduce new features that should be reviewed in the conda-build repository, not applied as patches.

File: recipe/gh3868.patch [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                     py3k=bool(30 <= py < 40),
+@@ -105,7 +105,11 @@ def ns_cfg(config):
+                     py33=bool(py == 33),
+                     py34=bool(py == 34),
+                     py35=bool(py == 35),
+-                    py36=bool(py == 36),))
++                    py36=bool(py == 36),
++                    py37=bool(py == 37),
++                    py38=bool(py == 38),
++                    py39=bool(py == 39),
```
Comment (jjhelmus) [inline recipe/gh3868.patch]: These selectors were explicitly deprecated in the conda build docs.

Comment (jjhelmus) [reply recipe/gh3868.patch]: From https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors

> The use of the Python version selectors, py27, py34, etc. is discouraged in favor of the more general comparison operators. Additional selectors in this series will not be added to conda-build.

**Comment by isuruf** [comment]
PR has been merged upstream. How about making this version `3.18.12.dev0`?

**Comment by isuruf** [comment]
ping

**Comment by scopatz** [comment]
@jjhelmus - do you still object?

**Comment by mingwandroid** [comment]
We are hoping to make a variety of new conda-build builds and a new release with this in ASAP.

**Comment by isuruf** [comment]
@mingwandroid, any updates?

**Comment by mingwandroid** [comment]
We will do a tag in a few hours then look into a point release soon after (next few days), sorry about the delay.

**Comment by isuruf** [comment]
Thanks for making the release

**Pull request closed without merging**